lib/bundlex/native.ex

```diff
@@ -158,6 +175,20 @@ defmodule Bundlex.Native do
     end
   end
 
+  defp filter_libs(libs, interface) do
+    libs
+    |> Enum.filter(fn lib ->
+      interfaces = Keyword.get(lib.config, :interface, []) |> Bunch.listify()
+
+      cond do
+        interfaces == [] -> true
+        interface in interfaces -> true
+        interface == nil && :nif in interfaces -> true
```

Comment:
This condition was for libs that do not specify an interface (backward compatibility) but want to use in deps some other lib that specify an interface e.g.
```elixir
      example_lib: [
        deps: [example_lib2: :example_lib2],
        src_base: ...
        sources: ...
      ]
    
      example_lib2: [
        deps: ...
        src_base:  ...
        sources:  ...
        interface: :nif
      ]
```

Do we want to support something like this ^?

Reply:
In this case `example_lib` will have the interface `nil` because it is not explicitly specified. We can also assume that lib which doesn't specify the interface should be threaten as `:nif` lib and insert `:nif` into its `interface` in `convert_input_config` but I am not sure if this will be backward compatible.

Reply:
> We can also assume that lib which doesn't specify the interface should be threaten as :nif lib

That's not possible, because from now a lib can be meant to specify no interface as well. Not sure that's possible to keep backward compatibility in this case, but don't think it's that important, so I'd drop it.

Reply:
done